packages/template-retail-react-app/app/translations/compiled/es-MX.json

```
@@ -1557,6 +1581,12 @@
       "value": "Ofertas"
     }
   ],
+  "lCPCxk": [
+    {
+      "type": 0,
+      "value": "Seleccine todas las opciones anteriores"
```

Not related to code changes, but you may want to let the translation team know there's a typo in the Spanish translation.
```suggestion
      "value": "Seleccione todas las opciones anteriores"
```

---

Haha yes Adam, you would know the correct translation for it 👍 I'll let them know, thanks.

---

haha, good catch Adam.